tasks/src/document-question-answering/about.md

```diff
@@ -0,0 +1,45 @@
+## Use Cases
+
+Document question answering models can be used to answer natural language questions directly from document images. This is useful when the visual information of the document is important for answering the question.
+
+### Document Parsing
+
+You can use Document question answering models to parse forms. For example, you can extract the name, address, and other information from a form. You can also use the model to extract information from a table, or even a resume.
+
+### Invoice Information Extraction
+
+You can use Document question answering models to extract information from invoices. For example, you can extract the invoice number, the invoice date, the total amount, the VAT number, and the invoice recipient.
+
+## Inference
+
+You can infer with Document QA models with the 🤗 Transformers library using the [`document-question-answering` pipeline](https://huggingface.co/docs/transformers/en/main_classes/pipelines#transformers.DocumentQuestionAnsweringPipeline). If no model checkpoint is given, the pipeline will be initialized with [`impira/layoutlm-document-qa`](https://huggingface.co/impira/layoutlm-document-qa). This pipeline answers the question(s) given as inputs by using the document(s). A document is defined as an image and an optional list of (word, box) tuples which represent the text in the document. If the word_boxes are not provided, it will use the Tesseract OCR engine (if available) to extract the words and boxes automatically for LayoutLM-like models which require them as input. For Donut, no OCR is run (see [paper](https://arxiv.org/abs/2111.15664)).
```

Comment:
Can you avoid from using passive voice? e.g. maybe simplify This pipeline answers the question(s) given as inputs by using the document(s).

Reply:
yes! sorry about that, hope it's better now